### PR TITLE
ATR-401: added missing nuget packages

### PR DESCRIPTION
**Pull request by SENya1990**

### CODE REVIEW

File: src/NuGet.Config [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
```
Comment (dropsonic) [inline src/NuGet.Config]: Do we need this?

Comment (SENya1990) [reply src/NuGet.Config]: It is autogenerated now. I tried to delete it but it was re-generated on build

Comment (SENya1990) [reply src/NuGet.Config]: I suppose it is part of "new format" for csproj file when you use embedded nuget references instead of package.config.

**Comment by SENya1990** [comment]
Try to build now

**Pull request merged**